ETC.tsv

```diff
@@ -12091,68 +12091,68 @@ ETC_20150714_012092	{memo X}Property:1/PropertyAdd:1/Func:SCR_REMAINS37_3_SQ_040
 ETC_20150714_012093	{memo X}Property:1/PropertyAdd:1/Notice:NOTICE_Dm_Clear:You've digged the gold plate well!:5/GiveItem:REMAINS37_3_SQ_040__ITEM_1:1
 ETC_20150714_012094	{memo X}DoTimeAction:Collecting:1.2:WEEDING:None
 ETC_20150714_012095	Collect the keepsakes of the advance party that is petrified
-ETC_20150714_012096	Experiment the symbols of Schilt
-ETC_20150714_012097	Collect the leaves of Bellas
-ETC_20150714_012098	Rub the juice of Bellas Grass Leaves on the crates of Amanda Tomb Robbers
-ETC_20150714_012099	Check the old market
-ETC_20150714_012100	Check the outer wall entrance
-ETC_20150714_012101	Eliminate the weak Stone Frosts at the great square
-ETC_20150714_012102	Rub the fragrant stone powders onto Rambear
-ETC_20150714_012103	Use Monster Stimulant
-ETC_20150714_012104	Lure the guard at Watermill Ruins
-ETC_20150714_012105	Put it at Estuary three ways
-ETC_20150714_012106	Put it at Zmone Square
-ETC_20150714_012107	Put it at Badova Dock
-ETC_20150714_012108	Put it at Viska Sanctum Ruins
+ETC_20150714_012096	$Test the Symbol of Silt
```

Comment:
I've asked IMC. Their reponse was that it didn't had any significant meaning. So, it's your pass.